code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -0,0 +1,84 @@
+///////////////File for all snowflake/special races/////////////////////////////
+/////Anything that is spectacularly special should be put in here///////////////
+////////////////////////////////////////////////////////////////////////////////
+
+
+
+/datum/species/xenochimera //Scree's race.
+	name = "Xenochimera"
+	name_plural = "Xenochimeras"
+	icobase = 'icons/mob/human_races/r_tajaran.dmi' //Tajara basis, as it has working skin color.
+	deform = 'icons/mob/human_races/r_def_tajaran.dmi'
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/punch, /datum/unarmed_attack/bite)
+	num_alternate_languages = 2
+	secondary_langs = list("Sol Common")
+	color_mult = 1
+	egg_type = "Scree" //Scree egg.
+	inherent_verbs = list(/mob/living/carbon/human/verb/chimera_revive)
+
+	min_age = 17
+	max_age = 80
+
+	blurb = "Some amalgamation of different species from across the universe,with extremely unstable DNA, making them unfit for regular cloners. \
+	Widely known for their voracious nature and violent tendencies when left unfed for long periods of time. \
+	Most, if not all chimeras possess the ability to undergo some type of regeneration process, at the cost of energy."
+
+
+	//primitive_form = "Farwa"
+
+	spawn_flags = SPECIES_CAN_JOIN || SPECIES_IS_WHITELISTED
```

Comment:
Single pipe, not double. Also we have whitelisting turned off so this needs to be RESTRICTED not WHITELISTED. The former means only admins can join as it.